__tests__/e2e/playwright.config.ts

```diff
@@ -43,6 +44,9 @@ export default defineConfig({
 	// Maximum time one test can run for.
 	timeout: 120_000,
 
+	// Snapshot path template to avoid having OS in name (difference CI vs local).
+	snapshotPathTemplate: "{testFileDir}/{testFileName}-snapshots/{arg}{ext}",
```

Comment:
Snapshots should not have the `{platform}` in their name, it's by default but will crash when running locally on OSX vs CI on Ubuntu